superset/utils/core.py

```diff
@@ -1550,7 +1550,7 @@ def override_user(user: Optional[User], force: bool = True) -> Iterator[Any]:
         delattr(g, "user")
 
 
-def parse_ssl_cert(certificate: str) -> _Certificate:
+def parse_ssl_cert(certificate: str) -> Certificate:
```

Comment:
nit: can you use `x509.Certificate` instead or remove the x509 import and import `load_pem_x509_certificate`

Reply:
good point, changed to the direct import of `load_pem_x509_certificate`